presentation-abstraction-control/src/test/java/com/iluwatar/pac/StudentStatusTest.java

```diff
@@ -0,0 +1,39 @@
+package com.iluwatar.pac;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class StudentStatusTest {
+  static StudentStatusControl studentStatusControl;
+  static StudentCourseControl studentCourseControl;
+//  @BeforeAll
+//  public static void start(){
+//    studentStatusControl=new StudentStatusControl();
+//    studentCourseControl=studentStatusControl.getStudentCourseControl();
+//    studentCourseControl.addCourseGrade("Math",90.0f);
+//    studentCourseControl.addCourseGrade("Physics",20.0f);
+//  }
```

Comment:
Remove the commented out code

Reply:
@iluwatar OK, I will refactor my code after my final exams and project presentation which really took over all my time